src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
```

Comment:
Do not use imitation as this is not the correct hierarchy, for example, the hierarchy: `person <- teacher` (you do not have to think about it, you will learn it in the next lessons)

**ToDo:** use instance of **ColorSupplier**, and [read this](https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery#think-about-which-variables-should-be-local-in-the-method-and-which-should-be-class-level)